src/Sulu/Bundle/AdminBundle/Resources/public/js/aura_extensions/sulu-content.js

```diff
@@ -136,17 +143,18 @@ define(function() {
          * Handles the navigation part of the layout object.
          *
          * @param {Object} navigation The navigation config object.
+         * @param {Boolean} defaults Iff true default actions will be executed
          */
-        handleLayoutNavigation = function(navigation) {
+        handleLayoutNavigation = function(navigation, defaults) {
```

Comment:
i think "useDefaukts"/"applyDefaults" would be a better name for the second parameter